CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+
+## Contributing
+
+We welcome all contributions!  Please head to [issues](https://github.com/Sage-Bionetworks/synapsemonitor/issues) to either file any bugs/feature requests or find a task you want to assist with.  Make sure to assign yourself the task if you decide to work on it.
+
+
+## Coding Style
+We use `flake8` and the settings are described in [setup.cfg](setup.cfg).  The code in this package is also automatically formatted by `black` except for the tests.
+
+
+### Fork and clone this repository
+
+See the [Github docs](https://help.github.com/articles/fork-a-repo/) for how to make a copy (a fork) of a repository to your own Github account.
+
+Then, [clone the repository](https://help.github.com/articles/cloning-a-repository/) to your local machine so you can begin making changes.
+
+Add this repository as an [upstream remote](https://help.github.com/en/articles/configuring-a-remote-for-a-fork) on your local git repository so that you are able to fetch the latest commits.
+
+On your local machine make sure you have the latest version of the `develop` branch:
+
+```
+git checkout develop
+git pull upstream develop
+```
+
+### Install development dependencies
+This will install all the dependencies of the package including the active branch of `synapsemonitor`.
+
+```
+pipenv install --dev
+```
+
+
+### The development life cycle
+
+`synapsemonitor` follows the standard [git flow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) development strategy.
+
+1. Pull the latest content from the `develop` branch of this central repository (not your fork).
+1. Create a feature branch which off the `develop` branch. If there is a GitHub issue that you are addressing, name the branch after the issue with some more detail (like `issue-123-add-some-new-feature`).
+1. After completing work and testing locally (see below), push to your fork.
+1. Make sure to run `black` the python code formatter: `black ./`
+1. In Github, create a pull request from the feature branch of your fork to the `develop` branch of the central repository.
```

Comment:
The language here is a little unclear to me.  Would suggest:

1. Example code for the relevant steps (1-4)
2. Making it explicit where each step is conducted and where changes occur (e.g. local forked repo, GitHub forked repo, GitHub source repo) 
3. Separating step 3 into three steps: make changes, run tests and push to GitHub forked repo